pages/curriculum/[lessonSlug]/[subLessonSlug].tsx

```diff
@@ -72,17 +144,10 @@ const SubLessonPage: React.FC<Props> & WithLayout = ({
         gridTemplateColumns: breakpoint ? 'auto auto' : 'none'
       }}
     >
-      <div
-        data-testid="toc"
-        className={`card shadow-sm border-0 ${styles.sublesson__sidebar}`}
-        style={{
-          position: breakpoint ? 'sticky' : 'initial'
-        }}
-      >
-        <ul className={`p-md-4 ${styles.toc__container} m-0 rounded`}>
-          {mapHeadingsToLi}
-        </ul>
-      </div>
+      <TableOfContents
+        selectedSubLesson={selectedSubLesson as SubLesson}
```

Comment:
Is type assertion required here too, in addtion to line 136?

Reply:
Yes. Without it, it would complain that the types don't match.

Reply:
After removing the type assertion, it doesn't complain. Nice catch, I'll update the code!